packages/vue/tests/app/README.md

```diff
@@ -0,0 +1,11 @@
+# Vue3 with GC Design System Components Vue Package
+
+This package is an example of how you can use the GC Design System Components - Vue package in a Vue3 project that uses Vite. Used to test component rendering, events listeners and `v-model`.
+
+# Vue 3 + Vite
+
+This template should help get you started developing with Vue 3 in Vite. The template uses Vue 3 `<script setup>` SFCs, check out the [script setup docs](https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup) to learn more.
+
+## Recommended IDE Setup
+
+- [VS Code](https://code.visualstudio.com/) + [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) (and disable Vetur) + [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin).
```

Comment:
Can we remove this file or change it to say that this is a test app used for e2e testing purposes?